doc/Programming-guide/Key-platform-features/Endpoint-profiles/index.md

```diff
@@ -406,3 +407,8 @@ The system part of the endpoint profile is used by Kaa internally for its functi
 |Endpoint Public Key      | The public key used for security purposes to validate endpoint requests           |
 |Endpoint group state     | The list of endpoint groups that contain current endpoint.                        |
 |Client-side Profile Hash | The client-side profile hash used to validate integrity of endpoint profile state |
+
+## Further reading
+
+* [Apache Avro](http://avro.apache.org/docs/current/spec.html)
```

Comment:
Further reading should contain only links to our docs.
Add link to endpoint grouping, configuration, notification, data collection features (they are affected by this article)